build_tools/git_version_gen.sh

```diff
@@ -34,7 +34,7 @@ if test -r $FBVF
 then
 	VC=$(grep -v '^#' $FBVF | tr -d '"' | sed -e 's/^FISH_BUILD_VERSION=//')
 else
-	VC=unset
+	VC=$(unset)
```

Comment:
Nope again. The former assigns the string `unset` to $VC, this runs the `unset` builtin?

Reply:
Correct. So it should be quoted. That's why shellcheck got confused.

Reply:
No it should not. It's a literal string, without any spaces or other special characters. It's entirely safe to not quote it.

Reply:
Sure. AFAIK, there's no real difference here. The quotes don't become literal here.

Reply:
Yeah, there's no difference. Which makes it super weird that shellcheck would fail this badly. This isn't a place for quotes (because the string is super simple) and this isn't a place where a command would be.